data/signersDefault.json

```diff
@@ -1,7 +1,5 @@
 {
 	"$schema": "./signersDefault-schema.json",
-	"versionTable": 1,
-	"versionData": 28,
```

Comment:
This information should still be needed in the old version for the update of the default rules to apply.

As the rules are updated, the `versionData` number needs to be increased to `29`.

I would also leave `data/signersDefault-schema.json` unchanged.

Reply:
Thank you for your explanations; I've simply took the files from the master branch, as they looked similar.
Now, I've merged the changes and incremented the rules version as requested.